packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -57,6 +58,12 @@ class Splitter extends (CollectionWidget as any) {
 
   _renderItems(items: Item[]): void {
     super._renderItems(items);
+
+    const splitterItemsCount = this._itemElements().length;
+    if (splitterItemsCount > 1) {
+      this.layoutHelper = new SplitterLayoutHelper(this._itemElements(), this.option('orientation'), this.$element());
```

Comment:
Should we work with visible items only and use _getVisibleItems()? Pane may be invisible.